modules/nodes-safe-sysctls-list.adoc

```diff
@@ -0,0 +1,123 @@
+For more information see link:https://docs.kernel.org/networking/ip-sysctl.html?highlight=ip_local_port_range#ip-variables[ip_local_port_range].
+
+| `net.ipv4.tcp_syncookies`
+|When net.ipv4.tcp_syncookies is set, the kernel will handle TCP SYN packets normally until the
+half-open connection queue is full, at which time, the SYN cookie functionality kicks in. This functionality SYN cookies allows the system to keep accepting valid connections, even if
+under a denial of service attack. For more information see link:https://docs.kernel.org/networking/ip-sysctl.html?highlight=tcp_syncookies#tcp-variables[tcp_syncookies].
+
+| `net.ipv4.ping_group_range`
+a| This restricts ICMP_PROTO datagram sockets to users in the group range. The default is “1 0”, meaning, that nobody (not even root) may create ping sockets. For more information see link:https://docs.kernel.org/networking/ip-sysctl.html?highlight=ping_group_range#ip-variables[ping_group_range].
+
+| `net.ipv4.ip_unprivileged_port_start`
+| This defines the first unprivileged port in the network namespace. To disable all privileged ports, set this to 0. They must not overlap with the ip_local_port_range. For more information see link:https://docs.kernel.org/networking/ip-sysctl.html?highlight=ip_unprivileged_port_start#ip-variables#ip-variables[ip_unprivileged_port_start].
+|===
+
+
+.Interface specific safe sysctls
+[cols="30%,70%",options="header"]
+|===
+| sysctl | Description
+
+| `net.ipv4.conf.IFNAME.accept_ra`
+a|Accept IPv4 Router Advertisements; autoconfigure using them. It also determines whether or not to transmit router solicitations. If and only if the functional setting is to accept Router Advertisements, Router Solicitations will be transmitted.
+
+| `net.ipv4.conf.IFNAME.accept_redirects`
+a| Accept IPv4 ICMP redirect messages.
+
+| `net.ipv4.conf.IFNAME.accept_source_route`
+|Accept IPv4 packets with strict source route (SRR) option.
+
+| `net.ipv4.conf.IFNAME.arp_accept`
+a| Define behavior for gratuitous ARP frames with an IPv4 address that is not already present in the ARP table:
+
+* 0 - do not create new entries in the ARP table
+
+* 1 - create new entries in the ARP table
+
+| `net.ipv4.conf.IFNAME.arp_notify`
+| Define mode for notification of IPv4 address and device changes.
+
+| `net.ipv4.conf.IFNAME.disable_policy`
+a| Disable IPSEC policy (SPD) for this IPv4 interface.
+
+| `net.ipv4.conf.IFNAME.secure_redirects`
+a| Accept ICMP redirect messages only to gateways listed in the interface’s current gateway list.
+
+| `net.ipv4.conf.IFNAME.send_redirects`
+| Send redirects, if router.
+
+| `net.ipv6.conf.IFNAME.accept_ra`
+a| Accept IPv6 Router Advertisements; autoconfigure using them. It also determines whether or not to transmit Router Solicitations. If and only if the functional setting is to accept Router Advertisements, Router Solicitations will be transmitted.
```

Comment:
```suggestion
a| Accept IPv6 Router Advertisements; autoconfigure using them. It also determines whether or not to transmit Router Solicitations. Router Solicitations will be transmitted only if the functional setting is to accept Router Advertisements.
```

Reply:
- Agree but "Router Solicitations will be transmitted" -> "Router Solicitations are transmitted"
- Are "Router Advertisements" and "Router Solicitations" proper nouns? Not sure they should be capitalized.